.github/workflows/deploy-master.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: build image
+        run: docker build -t documentation:latest .
```

Comment:
This can now be;

```suggestion
        run: DOCKER_BUILDKIT=1 docker build --build-arg=ENABLE_ARCHIVES=true --target=deploy-source --output=/output/_site .
```

Reply:
doesn't seem to work with builkit https://github.com/docker/docker.github.io/runs/466748285?check_suite_focus=true 🤔